pages/linux/addr2line.md

```diff
@@ -1,6 +1,7 @@
 # addr2line
 
 > Convert addresses of a binary into file names and line numbers.
+> More information: <https://www.gnu.org/software/binutils/>.
```

Comment:
```suggestion
> More information: <https://manned.org/addr2line>.
```
<https://www.gnu.org/software/binutils/> doesn't really give any information on the command.